### PR TITLE
Promise.any

### PR DESCRIPTION
**Pull request by lumosmind**
I think `Promise.any()` must be in this article. It is very critical when we work with multiple source content.

### CODE REVIEW

File: 1-js/11-async/05-promise-api/article.md [hunk 1]
```diff
@@ -217,6 +217,29 @@ Promise.race([
...
+
+```js run
+Promise.race([
+  new Promise((resolve, reject) => setTimeout(() => reject(new Error("Whoops!")), 1000)),
+  new Promise((resolve, reject) => setTimeout(() => resolve(1), 2000)),
+  new Promise((resolve, reject) => setTimeout(() => resolve(3), 3000))
+]).then(alert); // 1
+```
+
+The first promise here was fastest, but it rejecets, so the second promise became the result. After the first fulfilled promise "wins the race", all further results are ignored.
```
Comment (vsemozhetbyt) [inline 1-js/11-async/05-promise-api/article.md:240]:
```suggestion
The first promise here was fastest, but it was rejected, so the second promise became the result. After the first fulfilled promise "wins the race", all further results are ignored.
```

File: 1-js/11-async/05-promise-api/article.md [hunk 2]
```diff
@@ -217,6 +217,29 @@ Promise.race([
...
+The syntax is:
+
+```js
+let promise = Promise.any(iterable);
+```
+
+For instance, here the result will be `1`:
+
+```js run
+Promise.race([
```
Comment (iliakan) [inline 1-js/11-async/05-promise-api/article.md:233]: Promise.race example here?

**Review by iliakan — CHANGES_REQUESTED** [review]
Please fix the example.
And maybe add it to the Summary section?

File: 1-js/11-async/05-promise-api/article.md (hunk 2 above)
Comment (lumosmind) [reply 1-js/11-async/05-promise-api/article.md]: I did some changes to meet requirements

**Comment by iliakan** [comment]
Thank you!

**Pull request merged**